test/spmd/tensor/test_matrix_ops.py

```diff
@@ -3,6 +3,7 @@
 from torch.testing._internal.common_utils import run_tests
 from torch.testing._internal.common_distributed import skip_if_lt_x_gpu
 from spmd.tensor.api import DTensor
+from spmd.tensor.redistribute import redistribute_spmd_tensor
```

Comment:
any reason we use this function instead of directly `dc.redistribute`?

Reply:
not really, i can change to that .

Reply:
yep, `dc.redistribute` is the public API and it deals with the bwd pass as well